src/views/CharacterListView.js

```diff
@@ -1,35 +1,175 @@
-import React from "react";
-import { connect } from "react-redux";
+import React from 'react';
+import { connect } from 'react-redux';
+import { fetchData } from './../actions/index';
+import { CharacterList } from '../components';
+import styled from 'styled-components';
+import {
+	FlexFunc,
+	color_light,
+	header_font,
+	text_font,
+	shadow,
+	Button,
+	color_transparent,
+	color_background
+} from './../~reusables/reusables';
 
-import { CharacterList } from "../components";
-// import actions
+const CharListWrapper = styled.div`
+	${FlexFunc('column', 'center', 'center')};
+	color: ${color_light};
+	font-family: ${text_font};
+`;
+const CharListHeader = styled.h1`
+	font-size: 4rem;
+	font-family: ${header_font};
+	text-shadow: ${shadow};
+`;
+const SpinnerDiv = styled.div`
+	.lds-spinner {
```

Comment:
You could make this div fill the page and make the spinner central, something like ```min-height: 100vh``` and then use flex-box